docs/source/testing_services.rst

```diff
@@ -0,0 +1,205 @@
+| ``log_record``        | Any number                                                                                       | {"type": "log_record": "content": {"msg": "Starting analysis."}}                                                          |
++-----------------------+--------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| ``monitor_message``   | Any number                                                                                       | {"type": "monitor_message": "content": {"progress": "35%"}}                                                               |
++-----------------------+--------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| ``exception``         | One (technically, any number, but only the first will be raised)                                 | {"type": "exception", "content": {"exception_type": "ValueError", "exception_message": "x cannot be less than 10."}}      |
++-----------------------+--------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| ``result``            | One (technically, any number, but only the first will be returned)                               | {"type": "result", "content": {"output_values": {"my": "results"}, "output_manifest": None}}                              |
++-----------------------+--------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+
+
+**Notes**
+
+- Any messages after a ``result`` or ``exception`` message won't be passed to the parent because execution of the child
+  emulator will have ended.
+- The ``content`` of a ``log_record`` message is any dictionary that the ``logging.makeLogRecord`` function can convert
+  into a log record.
+- Message formats and contents are validated by :mod:`ChildEmulator <octue.cloud.emulators.child.ChildEmulator>`
+
+Input/output validation (lack of)
+---------------------------------
+Unlike a real child, the inputs given to the emulator and the outputs returned aren't validated against the schema in
+the child's twine - this is only available to the real child. Hence, the input values and manifest do not affect the
+messages returned by the emulator, and the output values and manifest are exactly what you specify in the messages you
+give to the emulator.
```

Comment:
The first sentence make sense but the second is overly complex. I think this could be simplified, as well as expanded into a more useful section about the creation of test fixtures

Creating test fixtures
---------------------

Since the child (or children) are *emulated*, that means they don't actually do any calculation - if you change the input values, the outputs won't change correspondingly. So it's up to you to define a set of realistic inputs and corresponding outputs. This is called a **test fixture**.

.. informationBoxWhateverTheSyntaxIsYouGetMe::
  Unlike a real child, the inputs given to the emulator and the outputs returned aren't validated against the schema in the child's twine, which is only available to the real child. That's ok, we're testing your service here, not the child!
  
Now we need a paragraph or two about how to create a test fixture in the first place!

Reply:
When you say test fixture, do you mean this group of things?
- Configuration values
- Configuration manifest
- Input values
- Input manifest